drivers/i3c/i3c_shell.c

```diff
@@ -86,6 +86,7 @@ struct i3c_ctrl {
 
 DT_FOREACH_STATUS_OKAY(cdns_i3c, I3C_CTRL_FN)
 DT_FOREACH_STATUS_OKAY(nxp_mcux_i3c, I3C_CTRL_FN)
+DT_FOREACH_STATUS_OKAY(nuvoton_npcx_i3c, I3C_CTRL_FN)
```

Comment:
nit: This should be in alphabetical order

Reply:
Done